pkg/asset/ignition/bootstrap/bootstrap.go

```diff
@@ -234,23 +236,68 @@ func (a *Bootstrap) addSystemdUnits(uri string, templateData *bootstrapTemplateD
 	}
 
 	for _, childInfo := range children {
-		name := childInfo.Name()
-		file, err := data.Assets.Open(path.Join(uri, name))
+		dir := path.Join(uri, childInfo.Name())
+		file, err := data.Assets.Open(dir)
 		if err != nil {
 			return err
 		}
 		defer file.Close()
 
-		name, data, err := readFile(name, file, templateData)
+		info, err := file.Stat()
 		if err != nil {
 			return err
 		}
 
-		unit := igntypes.Unit{Name: name, Contents: string(data)}
-		if _, ok := enabled[name]; ok {
-			unit.Enabled = util.BoolToPtr(true)
+		if info.IsDir() {
```

Comment:
Can we add a check here that says "if the directory name doesn't end with `.d` we ignore it (possibly with a debug-level log)?  From [the systemd docs][1], it looks like they require all drop-in directories to have a `.d` suffix.  But I'm not all that familiar with systemd, maybe I'm just misreading the docs.

[1]: https://www.freedesktop.org/software/systemd/man/systemd.unit.html

Reply:
I'm going to add a trace-level log line for that case. Trace should really only be used by developers which seems appropriate for this kind of message. Should I mention the "trace" level in the help output, or just keep that goody for ourselves?

Reply:
> Should I mention the "trace" level in the help output, or just keep that goodie for ourselves?

I'd mention it.  Users should recognize it as "the level after debug" even if they're non-devs, so I don't expect it will be too distracting.